app/src/main/java/org/schabi/newpipe/player/BackgroundPlayer.java

```diff
@@ -351,8 +353,12 @@ public void onUpdateProgress(int currentProgress, int duration, int bufferPercen
             resetNotification();
             if(Build.VERSION.SDK_INT >= 26 /*Oreo*/) updateNotificationThumbnail();
```

Comment:
I just saw this line while reviewing your PR and didn't want to send a new PR for this single change. Please add it :)
```suggestion
            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O /*Oreo*/) updateNotificationThumbnail();
```